### PR TITLE
Update readme

### LINKED ISSUE #246: Update contracts' README to use `pip install openzeppelin-cairo-contracts`

### PR DESCRIPTION
**Pull request by martriay**
Fixes #246

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,6 +19,109 @@ brew install gmp # mac
...
+from openzeppelin.token.erc20.ERC20 import constructor
+```
+
+Compile and deploy it right away:
+```bash
+nile compile
+
+nile deploy MyToken <name> <symbol> <decimals> <initial_supply> <recipient> --alias my_token
+```
+
```
Comment (andrew-fleming) [inline README.md:64]:
```suggestion

> Note that `<initial_supply>` is expected to be two integers i.e. `1` `0`. See [Uint256](docs/Utilities.md#Uint256) for more information.

```

File: README.md [hunk 2]
```diff
@@ -95,30 +196,15 @@ tests/test_Initializable.py .                              [ 89%]
...
-
 ## Security
 
 This project is still in a very early and experimental phase. It has never been audited nor thoroughly reviewed for security vulnerabilities. Do not use in production.
 
 Please report any security issues you find to security@openzeppelin.org.
 
+## Contribute
+OpenZeppelin Cairo Contracts exists thanks to its contributors. There are many ways you can participate and help build high quality software. Check out the [contribution](CONTRIBUTING.md) guide!
+
```
Comment (andrew-fleming) [inline README.md:207]: 👍

**Review by andrew-fleming — APPROVED** [review]
Very nice! I left a minor suggestion, but this looks good to go :)

File: README.md [hunk 3]
```diff
@@ -95,30 +198,15 @@ tests/test_Initializable.py .                              [ 89%]
...
-* [How to manage smart contract deployments with Nile](https://medium.com/@martriay/manage-your-starknet-deployments-with-nile-%EF%B8%8F-e849d40546dd)
-
 ## Security
 
 This project is still in a very early and experimental phase. It has never been audited nor thoroughly reviewed for security vulnerabilities. Do not use in production.
 
 Please report any security issues you find to security@openzeppelin.org.
 
+## Contribute
+OpenZeppelin Cairo Contracts exists thanks to its contributors. There are many ways you can participate and help build high quality software. Check out the [contribution](CONTRIBUTING.md) guide!
```
Comment (martriay) [inline README.md:208]:
```suggestion
OpenZeppelin Contracts for Cairo exists thanks to its contributors. There are many ways you can participate and help build high quality software. Check out the [contribution](CONTRIBUTING.md) guide!
```

**Pull request merged**